jetty-documentation/src/main/asciidoc/administration/sessions/session-configuration-sessioncache.adoc

```diff
@@ -72,4 +79,21 @@ The `NullSessionCache` is a trivial implementation of the `SessionCache` that do
 You may need to use it if your clustering setup does not have a sticky load balancer, or if you want absolutely minimal support for sessions.
 If you use this in conjunction with the `NullSessionDataStore`, then sessions will neither be retained in memory nor persisted.
 
-To enable the `NullSessionCache`, enable the `sesssion-cache-null` link:#startup-modules[module].
+To enable the `NullSessionCache`, enable the `sesssion-cache-null` link:#startup-modules[module]. 
+Configuration options are:
+
+jetty.session.saveOnCreate::
+Boolean, default `false`.
+Controls whether a session that is newly created will be immediately saved to the `SessionDataStore` or lazily saved as the last request for the session exits.
+
+jetty.session.removeUnloadableSessions::
+Boolean, default `false`.
+Controls whether a session that cannot be restored - for example because it is corrupted - from the `SessionDataStore` is deleted by the `SessionDataStore`.
+
+jetty.session.flushOnCommit::
+Boolean, default `false`.
```

Comment:
I thought this property defaulted to `true` since it makes the `HttpSession` behavior more sane?
I'm good with either value.

Reply:
I made the default false because:
+ it keeps the existing behaviour in a point release
+ we've only ever had 1 bug report related to this area, and that was a corner case

That said, if you want me to make default true, I'll do it.

Reply:
I'm fine with default at `false`.